scripts/swarm-fix-route.sh

```diff
@@ -507,9 +524,9 @@ function main {
 
       d)
       echo "STEP 8:"
-      echo "Removing NAT UDP 514 exceptions in <<$INGRESS_NS>> namespace..."
+      echo "Removing NAT UDP/514 UDP/162 TCP/6514 exceptions in <<$INGRESS_NS>> namespace..."
       delete_514_nat_exception $INGRESS_NS
-      echo "Removing NAT UDP 514 exceptions in <<$LB_NS_ID>> namespace..."
+      echo "Removing NAT UDP/514 UDP/162 TCP/6514 exceptions in <<$LB_NS_ID>> namespace..."
```

Comment:
echo "Removing NAT UDP/514 UDP/162 TCP/514 TCP/6514 exceptions in <<$LB_NS_ID>> namespace..."